### ISSUE TITLE
emlog 6.0 has SQL injection vulnerability

### ISSUE THREAD

**Issue by playZG** [emlog/emlog#144]
SQL injection exists in background batch deletion label，the passed in key was not verified
![5](https://user-images.githubusercontent.com/55383895/149527484-548a213e-c129-433a-bc54-fc4c5ffc8621.png)

Eventually, the $TagID parameter of getblogidsfromtagid() can be passed in malicious code
Function generating vulnerability
```php
    function getBlogIdsFromTagId($tagId) {
        $blogs = array();

        $sql = "SELECT `gid` FROM `" . DB_PREFIX . "tag` WHERE `tid` = " . $tagId;
        $query = $this->db->query($sql);

        if ($this->db->num_rows($query) > 0) {
            $result = $this->db->fetch_array($query);

            if ( ! empty($result['gid']))
            {
                $blogs = explode(',', $result['gid']);
            }
        }

        return $blogs;
    }
```

Verification
![4](https://user-images.githubusercontent.com/55383895/149527456-dd627dde-ab4b-4477-8ec4-7a383f744b1d.png)

![Inked3_LI](https://user-images.githubusercontent.com/55383895/149527601-b2501dc7-6200-412e-adce-d5eb7b0cadce.jpg)

![2](https://user-images.githubusercontent.com/55383895/149527741-873f3721-baf7-46f4-b888-91c0174e0f81.png)
![Inked1_LI](https://user-images.githubusercontent.com/55383895/149527756-b6b6aaa3-ad8c-4e1f-81f2-de55b64895a0.jpg)

**Comment by emlog** [comment]
how to hack into background?

**Comment by playZG** [comment]
> 如何入侵后台？

I think blasting is a way, but I haven't tried it yet

**Comment by playZG** [comment]
> how to hack into background?

I think blasting is a way, but I haven't tried it yet

**Comment by emlog** [comment]
ok thx i will fix it in emlog pro, 6.0 is stop maintenance
